SimpleRabbit.NetCore/Model/RabbitConfiguration.cs

```diff
@@ -1,15 +1,15 @@
-using System;
 using System.Collections.Generic;
 
 namespace SimpleRabbit.NetCore
 {
     public class RabbitConfiguration
     {
-        public Uri Uri { get; set; }
         public List<string> Hostnames { get; set; }
+        
+        public string Protocol { get; set; }

```

Comment:
NIT: Protocol is currently unused, Connection Factory is defaulted to amqp, Is this needed?